packages/docs/docs/guide/i18n.md

```diff
@@ -58,6 +58,10 @@ module.exports = {
         ariaLabel: 'Languages'
         // text for the edit-on-github link
         editLinkText: 'Edit this page on GitHub',
+        // text for the 404 page link
+        notFoundLinkText: 'Take me home.',
+        // the list of random messages to show on the 404 page
+        notFoundMessages: [/* ... */],
```

Comment:
I think this is important to write an example with the default values here, to let the user know what the default values are.